server/src/main/java/io/deephaven/server/table/ops/ColumnStatisticsGrpcImpl.java

```diff
@@ -68,19 +67,12 @@ public Table create(ColumnStatisticsRequest request, List<SessionState.ExportObj
         // Based on the column type, make a stats function and get a column source
         final ChunkedStatsKernel statsFunc;
         final ColumnSource<?> columnSource;
-        if (DateTimeUtils.isDateTime(type)) {
-            // Instant/ZonedDateTime only look at max/min and count
+        if (type == Instant.class) {
             statsFunc = new DateTimeChunkedStats();
-
-            // Reinterpret the column to read only long values
-            if (Instant.class.isAssignableFrom(type)) {
-                columnSource = ReinterpretUtils.instantToLongSource(table.getColumnSource(columnName));
-            } else if (ZonedDateTime.class.isAssignableFrom(type)) {
-                columnSource = ReinterpretUtils.zonedDateTimeToLongSource(table.getColumnSource(columnName));
-            } else {
-                throw Exceptions.statusRuntimeException(Code.INVALID_ARGUMENT,
-                        "DateTime columns must be Instant or ZonedDateTime");
-            }
+            columnSource = ReinterpretUtils.instantToLongSource(table.getColumnSource(columnName));
+        } else if (type == ZonedDateTime.class) {
+            statsFunc = new DateTimeChunkedStats();
+            columnSource = ReinterpretUtils.zonedDateTimeToLongSource(table.getColumnSource(columnName));
```

Comment:
Fun fact, this is going to work, but it has implications. That is, it assumes that all ZDTs that map to the same Instant are equal, which isn't so according to ZDT.equals(). Not a thing to address here, now, but certainly a concern.